### DISCUSSION TITLE
Lens not working anymore

### DISCUSSION THREAD

**Discussion by AliMohameddddd** [ptrumpis/snap-camera-server#40] [Q&A] [answered]
I've noticed that some of the lenses in the application are no longer available, and I'm curious about the reason behind this. Initially, these lenses worked perfectly when I was using the older version. However, over time, I encountered buffering issues with certain filters, prompting me to reinstall the app with the latest version. Unfortunately, after the update, some lenses seem to have disappeared entirely. If anyone has any insights into the cause of this problem or any possible fixes, I would be incredibly grateful for your assistance.
Lens name :Clear skin by jaylynn a
Link: https://www.snapchat.com/lens/1d88d03318714f45a274a5fd92c2504d?sender_web_id=067d441a-2e0c-4869-8c9a-09246bc04c3d&device_type=desktop&is_copy_url=true

**Comment by ptrumpis** [comment, marked as answer]
Pasting this link in the search bar should work:
https://lens.snapchat.com/1d88d03318714f45a274a5fd92c2504d

If you have any other non working links, try to replace `https://www.snapchat.com/lens/` with `https://lens.snapchat.com/`

I will update the server code and fix the problem in the next version.

**Comment by AliMohameddddd** [comment]
The lens is in continuous loading and never actually work for some reason...Did you by any chance alter your sql database that it in longer have some filters?

**Reply by ptrumpis** [reply]
Did you apply the signature patch?
You said it was working before the update?

If you need assistance you have to follow the steps, fill out the information and submit this form

[Issue: Lens not working ](https://github.com/ptrumpis/snap-camera-server/issues/new?assignees=ptrumpis&labels=lens+not+working&projects=&template=lens-not-working.md&title=)
